### DISCUSSION TITLE
Rita html report integration

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2262] [Unsupported Versions]
In the 16.04 SO, I had a useful 7 days rolling rita automatically generating html report every day and serving it on <host>/rita with cron and lighttpd (if I remember).
With the new SO2, I can't yet find a way to serve that rita directory either with the included nginx or an extra lighttpd.
I've tried modifying the so-firewall and changing the sls to make lighttpd accessible over another port with no luck so far.
Any help would be appreciated.

**Comment by weslambert** [comment]
This isn't officially supported, but I'll try having a look as I get time, as I'd like to get procedures for custom integration more well-documented.

**Reply by 0nnyx** [reply]
Thanks for your reply Wes. Wasn't really expecting any as it's not a priority / strictly SO.

**Reply by Mastadamus** [reply]
Did you ever get this working Onnyx? I'm looking to do the same.

**Comment by 0nnyx** [comment]
I didn't try further as it's understandably not a priority for SO devs. Am currently using a workaround with zeek on my firewall and scheduled scp to a container running rita & apache + added an html link on the SO overview page

**Comment by vijaytejak** [comment]
Any workaround found for this? to push the RITA logs to Security Onion.
